File: app/src/main/java/com/bresiu/codechallenge/model/Photo.java
package com.bresiu.codechallenge.model;

public class Photo {
	public int id;
	public int albumId;
	public String title;
	public String url;
	public String thumbnailUrl;

	public Photo(int id, int albumId, String title, String url, String thumbnailUrl) {
		this.id = id;
		this.albumId = albumId;
		this.title = title;
		this.url = url;
		this.thumbnailUrl = thumbnailUrl;
	}
}

File: app/src/main/java/com/bresiu/codechallenge/db/entity/Photo.kt
package com.bresiu.codechallenge.db.entity

import android.arch.persistence.room.Entity
import android.arch.persistence.room.ForeignKey
import android.arch.persistence.room.PrimaryKey

@Entity(foreignKeys = [(ForeignKey(entity = Album::class, parentColumns = arrayOf("id"), childColumns = arrayOf("albumId"), onDelete = ForeignKey.CASCADE))])
class Photo {
    @PrimaryKey
    internal var id: Long = 0
    internal var albumId: Long = 0
    internal var title: String? = null
    internal var url: String? = null
    internal var thumbnailUrl: String? = null
}
